cfn-resources/util/util.go

```diff
@@ -286,6 +288,63 @@ func (c *Config) NewSDKV2Client(client *http.Client) (*atlasSDK.APIClient, error
 	return sdkV2, nil
 }
 
+// NewAtlasV2OnlyClientLatest creates atlas-go-sdk client with the most recent version of Atlas SDK
+func NewAtlasV2OnlyClientLatest(req *handler.Request, profileName *string, profileNamePrefixRequired bool) (*MongoDBClient, *handler.ProgressEvent) {
+	prof, err := profile.NewProfile(req, profileName, profileNamePrefixRequired)
+
+	if err != nil {
+		return nil, &handler.ProgressEvent{
+			OperationStatus:  handler.Failed,
+			Message:          err.Error(),
+			HandlerErrorCode: cloudformation.HandlerErrorCodeNotFound}
+	}
+
+	// setup a transport to handle digest
+	transport := digest.NewTransport(prof.PublicKey, prof.PrivateKey)
+
+	// initialize the client
+	client, err := transport.Client()
+	if err != nil {
+		return nil, &handler.ProgressEvent{
+			OperationStatus:  handler.Failed,
+			Message:          err.Error(),
+			HandlerErrorCode: cloudformation.HandlerErrorCodeInvalidRequest}
+	}
+
+	c := Config{BaseURL: prof.BaseURL}
+	// New SDK Client
+	sdkV2Client, err := c.NewSDKV2LatestClient(client)
+	if err != nil {
+		return nil, &handler.ProgressEvent{
+			OperationStatus:  handler.Failed,
+			Message:          err.Error(),
+			HandlerErrorCode: cloudformation.HandlerErrorCodeInvalidRequest}
+	}
+
+	clients := &MongoDBClient{
+		AtlasSDKLatest: sdkV2Client,
+		Config:         &c,
+	}
+
+	return clients, nil
+}
+
+func (c *Config) NewSDKV2LatestClient(client *http.Client) (*adminLatest.APIClient, error) {
+	opts := []adminLatest.ClientModifier{
+		adminLatest.UseHTTPClient(client),
+		adminLatest.UseUserAgent(userAgent),
+		adminLatest.UseBaseURL(c.BaseURL),
+		adminLatest.UseDebug(false)}
+
+	// Initialize the MongoDB Versioned Atlas Client.
+	sdkV2, err := adminLatest.NewClient(opts...)
+	if err != nil {
+		return nil, err
+	}
+
+	return sdkV2, nil
+}
+
```

Comment:
Do we want to update the examples to add new fields?

Reply:
good point! will add

Reply:
added